File: Java_ChildrenCompetition/src/main/java/model/TestDTO.java
package model;

public class TestDTO extends Entity<Integer> {
    private final String testType;
    private final String testAgeCategory;
    private int noCompetitors;

    public TestDTO(String testType, String testAgeCategory, int noCompetitors) {
        this.testType = testType;
        this.testAgeCategory = testAgeCategory;
        this.noCompetitors = noCompetitors;
    }

    public String getTestType() {
        return testType;
    }

    public String getTestAgeCategory() {
        return testAgeCategory;
    }

    public int getNoCompetitors() {
        return noCompetitors;
    }
}

File: CSharp_ChildrenCompetitionGUI/CSharp_ChildrenCompetitionGUI/CSharp_ChildrenCompetitionGUI/model/TestDTO.cs
namespace CSharp_ChildrenCompetitionGUI.model
{
    public class TestDTO : Entity<int>
    {
        public string testType { get; }
        public string testAgeCategory { get; }
        public int noCompetitors { get; }

        public TestDTO(string testType, string testAgeCategory, int noCompetitors)
        {
            this.testType = testType;
            this.testAgeCategory = testAgeCategory;
            this.noCompetitors = noCompetitors;
        }
    }
}